doc/source/examples/unresolved-cfd-dem/gas-solid-fluidized-bed-cylindrical/gas-solid-fluidized-bed-cylindrical.rst

```diff
@@ -0,0 +1,499 @@
+
+The figure on the left compares the matrix-based (MB) solver results with cell-based force projection for models A and B, as well as the matrix-free (MF) solver for model A (which uses only QCM filtering for the force projection). The figure on the right compares the MB solver results with QCM force projection for models A and B and the MF solver with QCM force projection.
+
+It can be seen that the increasing portion of the pressure-drop curve varies only slightly between the MB simulations with cell-based force projection and the MF model. In contrast, the MB simulations using QCM force projection exhibit a steeper rise before fluidization, resulting in a fluidization point closer to the shown correlations. This behaviour may be attributed to the smaller time step required for numerical stability when using an explicit drag coupling with QCM force projection, as well as to the cell-independent nature of the filtering (which is also the case for the MF solver). Discrepancies appear in the fluctuations of the plateau region after fluidization, which may be better resolved by using a smaller time step or tighter nonlinear solver tolerances.
+
+The following table summarizes the fluidization Reynolds number obtained with each simulation (taken to be the first intersection point with the horizontal dotted line), compared to the Wen-Yu [#WenYu1966]_  and Noda *et al* [#Noda1986]_ correlations:
+
+    +---------------------------+-------------+------------------+------------------------+
+    | Solver                    | VANS Model  | Force projection | :math:`Re_{\text{mf}}` |
+    +===========================+=============+==================+========================+
+    | Matrix-based              | A           | Cell-based       | 426                    |
+    +---------------------------+-------------+------------------+------------------------+
+    | Matrix-based              | B           | Cell-based       | 393                    |
+    +---------------------------+-------------+------------------+------------------------+
+    | Matrix-based              | A           | QCM              | 361                    |
+    +---------------------------+-------------+------------------+------------------------+
+    | Matrix-based              | B           | QCM              | 361                    |
+    +---------------------------+-------------+------------------+------------------------+
+    | Matrix-free               | A           | QCM              | 426                    |
+    +---------------------------+-------------+------------------+------------------------+
+    | Wen-Yu [#WenYu1966]_      | A           | QCM              | 270                    |
+    +---------------------------+-------------+------------------+------------------------+
+    | Noda *et al* [#Noda1986]_ | A           | QCM              | 294                    |
```

Comment:
I feel your analysis here is confusing some things.
The QCM matrix-based implementation should lead to similar/identical results as the matrix-free one if you use the same coupling strategy. I think here the most important difference is that one uses the explicit coupling and the other one uses a semi-implicit one. It would be more relevant to have a column that specifies the coupling strategy (semi-implicit, implicit, explicit). 

I'm not sure if it's relevant to do it for both model A and B and QCM/PCM because that leads to a very large number of simulations. But if you really want to compare them, you need to specify the coupling strategy and compare apples with apples, because right now this not very adequate.